src/cat.rs

```diff
@@ -68,41 +60,46 @@ pub fn start(cli: &ArgMatches<'_>) {
         Box::new(io::stdout())
     };
 
-    let name = get_name1(cli.value_of("PROFILE_NAME").unwrap());
-    let path = find_profile(&name).unwrap_or_else(|| fatal!("Can not find {}.", &name));
-    let data = read_to_string(&path)
-        .unwrap_or_else(|e| fatal!("Failed to open {}: {}", path.to_string_lossy(), e));
-
-    let profile = Profile { path, data };
-
     let opts = Options {
         show_locals: !cli.is_present("no-locals"),
         show_redirects: !cli.is_present("no-redirects"),
     };
+    let name = cli.value_of("PROFILE_NAME").unwrap();
+    let profile_flags = ProfileFlags::default();
+    let profile = Profile::new(name, profile_flags).unwrap();
 
-    process(&profile, &opts, &mut output, 0);
+    if let Some(path) = profile.path() {
+        let content = read_content(path).expect("Couldn't Read file.");
+        process(&profile, &content, &opts, &mut output, 0);
+    }
 
     drop(output); // We need to drop output here, otherwise we would have two mutable references.
     if let Some(ref mut child) = child {
         child.wait().unwrap();
     }
 }
 
-fn process<W: io::Write>(profile: &Profile, opts: &Options, output: &mut W, mut depth: u8) {
+fn process<W: io::Write>(
+    profile: &Profile,
+    content: &str,
+    opts: &Options,
+    output: &mut W,
+    mut depth: u8,
+) {
     if depth >= 16 {
-        fatal!("To many include levels");
+        error!("Too many include levels");
     }
     depth += 1;
 
-    let [locals, profiles] = parse(&profile.data);
+    let [locals, profiles] = parse(&content);
 
     if opts.show_locals {
         if let Some(locals) = locals {
             show_locals(&locals, opts, output);
         }
     }
 
-    show_file(profile, output);
+    show_file(&profile, &content, output);
```

Comment:
It's not necessary to add `&` to profile. because `profile` is already borrowed. However it does not harm.

Reply:
Noted.